info/release-notes/23.next.0-readme.txt

```diff
@@ -67,7 +67,9 @@ Changes
 Bugfix
 ======
 
-* 
+* Air Quality
```

Comment:
The tour info UI displays the saved airquality ID

![image](https://github.com/mytourbook/mytourbook/assets/1283445/6e85aba6-a680-40fd-b706-01e106544b9d)

Reply:
Good catch, I will fix that